dashboard/app.py

```diff
@@ -68,22 +68,42 @@ def main():
 
 @app.route('/leaderboard')
 def leaderboard():
-    page_content = cache.get('leaderboard')
+    page = int(request.args.get('page', 1))
+    if page < 1:
+        abort(400)
+
+    page_content = cache.get('leaderboard-page-{}'.format(page))
```

Comment:
Not quite related to PR, but our cache was moved somewhere and is currently deprecated:
<img width="993" alt="Screenshot 2019-04-03 at 12 19 48" src="https://user-images.githubusercontent.com/1409983/55467857-cd90ee80-560a-11e9-9057-454db4c23075.png">

Reply:
added #106